falcon/routing/static.py

```diff
@@ -66,7 +66,7 @@ def __init__(self, prefix, directory, downloadable=False, fallback_filename=None
             prefix += '/'
 
         self._prefix = prefix
-        self._directory = directory
+        self._directory = os.path.normpath(directory)
```

Comment:
Maybe this normpath should be before the check on line 52?
https://github.com/falconry/falcon/blob/e0459ebf7f85a65c1cafa968330985d60cbd036c/falcon/routing/static.py#L52-L53

Reply:
Yeah, I think that would make sense. From the `isabs()` docs:

> Return True if path is an absolute pathname. On Unix, that means it begins with a slash, on Windows that it begins with a (back)slash after chopping off a potential drive letter.

Reply:
fixed in the last commit